pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo</groupId>
+    <artifactId>nuxeo-parent</artifactId>
+    <version>2021.15</version>
+  </parent>
+
+  <groupId>org.nuxeo.compound.documents</groupId>
+  <artifactId>nuxeo-compound-documents-parent</artifactId>
+  <name>Nuxeo Compound Documents Parent</name>
+  <version>2021.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <description>Nuxeo Compound Documents allow binding several documents as parts of one complex document.</description>
+
+  <modules>
+    <module>nuxeo-compound-documents</module>
+    <module>nuxeo-compound-documents-package</module>
+  </modules>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>${project.groupId}</groupId>
+        <artifactId>nuxeo-compound-documents</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>${project.groupId}</groupId>
+        <artifactId>nuxeo-compound-documents-package</artifactId>
+        <version>${project.version}</version>
+        <type>zip</type>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <repositories>
+    <repository>
+      <id>public</id>
+      <url>https://packages.nuxeo.com/repository/maven-public</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <updatePolicy>always</updatePolicy>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
```

Comment:
Can be removed, https://packages.nuxeo.com/repository/maven-internal/ is a top-level group including [maven-public](https://packages.nuxeo.com/repository/maven-public).